### PR TITLE
Update postMessage(powerboxRequest) documentation to match #1675

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: src/sandstorm/grain.capnp [hunk 1]
```diff
@@ -655,9 +655,9 @@ interface SessionContext {
   # powerboxRequest:
   #   rpcId: A unique string that should identify this rpc message to the app. You will receive this
   #          id in the callback to verify which message it is referring to.
-  #   query: A list of PowerboxDescriptor objects, serialized as a Javascript array OR a
-  #          base64-encoded powerbox query created using the `spk query` tool.
```
Comment (gkoz) [inline src/sandstorm/grain.capnp]: I've failed to find any traces of `spk query` command.

Comment (kentonv) [inline src/sandstorm/grain.capnp]: This is something we plan to add but I suppose we shouldn't have it in the doc if it's not there yet!

**Comment by paulproteus** [comment]
Since when do I own the comments in *.capnp files?

I'm OK owning them, I guess, though!

**Comment by kentonv** [comment]
Wow, for some reason I thought this was docs. Apparently I didn't look very hard.

**Comment by kentonv** [comment]
In that case I'd prefer to keep the `spk query` comment because frankly a lot of the comments in this file describe things that don't exist yet.

But the other fixes look good.

**Comment by gkoz** [comment]
Restored the mention of `spk query`.

**Comment by kentonv** [comment]
Thanks!

**Comment by paulproteus** [comment]
BTW, I'm +1 on *not* owning the comments in capnp files. (-: Thanks for
carefully reviewing this, Kenton, and merging it!

**Pull request merged**